cinema/urls.py

```diff
@@ -1,10 +1,39 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    CinemaHallViewSet,
+    ActorDetail,
+    ActorList,
+    MovieViewSet,
+    GenreDetail,
+    GenreList,
+)
 
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
+cinema_hall = CinemaHallViewSet.as_view(
+    actions={
+        "get": "list",
+        "post": "create",
+    }
+)
+cinema_detail = CinemaHallViewSet.as_view(
+    actions={
+        "get": "retrieve",
+        "put": "update",
+        "patch": "partial_update",
+        "delete": "destroy",
+    }
+)
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("", include(router.urls)),
```

Comment:
No need to include if basepath not given
urlpatterns = [list of paths] + router.urls